packages/strapi-design-system/src/SimpleNav/SimpleNav.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Flex } from '../Flex';
+
+export const SimpleNav = (props) => {
+  return <Flex {...props} />;
+};
+
+SimpleNav.defaultProps = {
+  ...Flex.defaultProps,
+  as: 'nav',
+  display: 'flex',
```

Comment:
IMO we shouldn't be giving users the options to change these styles _easily_ of course, they can wrap in their own `styled` components, but in the interest of a small API surface I think we should just keep it simple. 

I don't think we should spread all the props in the component so it could be as simple as:

```jsx
export const SimpleNav = (props) => {
	return <Flex>{props.children}</Flex>
}
```

Unless we explicitly _need_ users to add more props in which case they should be added.